### PR TITLE
feat: takeoff issues command

### PR DESCRIPTION
**Pull request by ImenesFBW**
## Description

 Provides help regarding engine issues on takeoff

## Test Results

image

## Discord Username

Imenes#0320

### CODE REVIEW

File: src/commands/a32nx/takeoffissues.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export const takeoffIssues: CommandDefinition = {
+    name: ['takeoffissues', 'toi', 'engines', 'eng'],
+    description: 'Help with engine issues on takeoff',
+    category: CommandCategory.A32NX,
+    executor: async (msg) => {
+        const contentEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | Engine Issues on Takeoff',
+            description: makeLines([
+                'Engines shutting down on takeoff is caused by \'ENGINE AUTOSTOP\' and \'AUTO START ENGINE\' bindings, make sure you unbind those in the controls options.',
```
Comment (oim1) [inline src/commands/a32nx/takeoffissues.ts:15]:
```suggestion
                'Engines shutting down on takeoff is caused by the \'ENGINE AUTOSTOP\' and \'AUTO START ENGINE\' bindings. Please make sure that you unbind these in the controls options
```

**Review by oim1 — CHANGES_REQUESTED** [review]
everything else looks fine, we can discuss wording in the discord server

**Comment by ImenesFBW** [comment]
> everything else looks fine, we can discuss wording in the discord server

Your wording is better + the missing "the". Committed your suggestion. Thanks!

**Review by oim1 — APPROVED** [review]
lgtm, will get ben to do a final proof read in case my grammar sucks

**Review by benw202 — CHANGES_REQUESTED** [review]
lgtm!

**Review by benw202 — APPROVED** [review]

**Pull request merged**